### PR TITLE
core: lpae: zero initialize all l1 tables

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Fixes: 4cc2823eb25c ("core: user mode translation table")
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>
Tested-by: Jens Wiklander <jens.wiklander@linaro.org> (Hikey)

### CODE REVIEW

File: core/arch/arm/mm/core_mmu_lpae.c [hunk 1]
```diff
@@ -509,7 +509,7 @@ void core_init_mmu_tables(struct tee_mmap_region *mm)
 	}
 
 	/* Clear table before use */
-	memset(l1_xlation_table[0], 0, NUM_L1_ENTRIES * XLAT_ENTRY_SIZE);
+	memset(l1_xlation_table, 0, sizeof(l1_xlation_table));
```
Comment (etienne-lms) [inline core/arch/arm/mm/core_mmu_lpae.c]: remove the part below (the `for` loop).

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by jforissier** [comment]
`Tested-by: Jerome Forissier <jerome.forissier@linaro.org> (HiKey)`

**Comment by jenswi-linaro** [comment]
Tags applied

**Pull request merged**